pipelines/common/src/test/resources/sql-on-fhir-v2-tests/basic.json

```diff
@@ -2,9 +2,9 @@
   "title": "basic",
   "description": "basic view definition",
   "fhirVersion": [
-    "5.0",
-    "4.0",
-    "3.0"
+    "5.0.0",
+    "4.0.1",
```

Comment:
Question: Looks like we only use the first two digits in the version i.e. in the `fhirVersion` format `x.y.z` we only use `x` and `y` to determine the version and `z` is ignored. Should we also check if the `z` is supported

Reply:
Correct, there was some discussion around 2 vs 3 segment version numbers in [this thread](https://github.com/FHIR/sql-on-fhir-v2/pull/236#discussion_r1584158825). Eventually I ended up with 3 segment versions but HAPI doesn't have separate FHIR context/FHIRPath engine for all versions. So we only care about the official releases, which I think would be the vast majority of the cases (but I don't want the code to fail for other version numbers, just because of version check). That's why we only care about the first 2 digits.

Reply:
Ok. May be we can merge this change, post the merge of this other [PR](https://github.com/FHIR/sql-on-fhir-v2/pull/236). WDYT?

Reply:
Sure, I'll merge this after; that PR is probably going to be merged very soon (I may do it myself after the meeting today).